### PR TITLE
Added apsimx package

### LINKED ISSUE #7974: Uninstalled apsimx package.
I am working on a new dialog (apsimx examples) for crop simulation. This dialog needs the apsimx package to be able to run its models package which is not yet installed in R-instat. Please can this package be part of the pre-installed package?  

image

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes #7974.
@N-thony and @lloyddewit please can you have a look? Thank you.

### CODE REVIEW

**Review by N-thony — APPROVED** [review]
@MeSophie great, @lloyddewit over to you

**Comment by N-thony** [comment]
@MeSophie can you resolve the conflicts?

**Comment by MeSophie** [comment]
> @MeSophie can you resolve the conflicts?

@N-thony  it is okay now.

**Comment by MeSophie** [comment]
> @rdstern I think the package need in #7963 can be included in this PR. @MeSophie we can have a quick chat on how to add it.

@N-thony okay

**Comment by MeSophie** [comment]
> @rdstern I think the package need in #7963 can be included in this PR. @MeSophie we can have a quick chat on how to add it.

@N-thony I'm done

**Review by rdstern — DISMISSED** [review]
@MeSophie I am not sure what I can test here?  I am approving, because the examples still work for me - but I installed apsimx ages ago anyway.

File: instat/static/InstatObject/R/install_packages.R [hunk 1]
```diff
@@ -130,12 +130,13 @@ pkgs <-
     # For most frequent values in Prepare > Column:Numeric > Row Summaries
     "statip",
     # also install mmtable2 from GitHub devtools::install_github("ianmoran11/mmtable2")
+    # install.packages('remotes')
+    # also install aida-package from GitHub remotes::install_github('michael-franke/aida-package')
     "corrr",
     "dslabs",
-    "coin",
-    "pingr",
-    "vcdExtra"
```
Comment (lloyddewit) [inline instat/static/InstatObject/R/install_packages.R:137]: Issue #7974 does not mention these pacakages. What's the reason for removing them?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@MeSophie Thanks for adding this pacakage. One open question.

File: instat/static/InstatObject/R/install_packages.R (hunk 1 above)
Comment (MeSophie) [reply instat/static/InstatObject/R/install_packages.R]: > Issue #7974 does not mention these pacakages. What's the reason for removing them?

@lloyddewit I didn't notice it. I did it by mistake.

File: instat/static/InstatObject/R/install_packages.R [hunk 2]
```diff
@@ -130,12 +130,16 @@ pkgs <-
     # For most frequent values in Prepare > Column:Numeric > Row Summaries
     "statip",
     # also install mmtable2 from GitHub devtools::install_github("ianmoran11/mmtable2")
+    # install.packages('remotes')
+    # also install aida-package from GitHub remotes::install_github('michael-franke/aida-package')
     "corrr",
     "dslabs",
-    "coin",
+     "coin",
```
Comment (lloyddewit) [inline instat/static/InstatObject/R/install_packages.R:137]:
```suggestion
    "coin",
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**